test/routes/items_spec.js

```diff
@@ -0,0 +1,99 @@
+const expect = require( 'chai' ).expect
+const sinon = require( 'sinon' )
+const express = require( 'express' )
+
+// const stubbedResponse = ( expected, done ) => {
+//   return {
+//     render: data => {
+//       expect( data ).to.be.eql( expected )
+//       done()
+//     }
+//   }
+// }
+
+describe( 'items routes', () => {
+  let sandbox
+  let router
+
+  beforeEach( () => {
+    sandbox = sinon.sandbox.create()
+    sandbox.stub( express, 'Router' ).returns({
+      get: sandbox.spy(),
+      post: sandbox.spy(),
+      delete: sandbox.spy()
+    })
+
+    router = require( '../../routes/items' )
+  })
+
+  it( 'registers URI / for GET', () => {
+    expect( router.get.calledWith( '/', sandbox.match.any )).to.be.true
+  })
+
+  describe( 'GET /starred URI', () => {
+    it( 'is registered', () => {
+      expect( router.get.calledWith( '/starred', sandbox.match.any )).to.be.true
+    })
+
+    it( 'renders the things', done => {
```

Comment:
@jrob8577 Totally! Monica and I were discussing what the test is actually doing. She suspects that it is testing rendering, and I think that "returns json data" might be a good name. Thoughts?